lib/cisco_node_utils/DELETED.command_reference_common.yaml

```diff
@@ -1053,39 +1053,3 @@ yum:
     config_get_token: '/^%s\S*\s+(\S+)\s+(?:installed|@\S+)/'
   remove:
     config_set: "install deactivate %s"
```

Comment:
I believe so. The definitions for vxlan_vtep, vxlan, pim etc exist there and they don't have their own yaml files yet. The node utils for these needs to be developed.